src/Fhir.Metrics/Utils/Parser.cs

```diff
@@ -51,45 +52,53 @@ public static class Parser
     {
 
         public readonly static Regex TokenPattern = new Regex(@"(((?<m>[\.\/])?(?<m>[^\.\/]+))*)?", RegexOptions.Compiled);
-        public readonly static Regex Annotations = new Regex(@"{[^{}]*}", RegexOptions.Compiled);
+        public readonly static Regex ValidAnnotationsPattern = new Regex(@"{[^{}]+}(?=$|/|\.)", RegexOptions.Compiled);
```

Comment:
To keep it consistent as **TokenPattern**, I would suggest to escape the forward slash, although technically it is not needed here.

```suggestion
        public readonly static Regex ValidAnnotationsPattern = new Regex(@"{[^{}]+}(?=$|\/|\.)", RegexOptions.Compiled);
```